terra-base-jupyter/custom/README.md

```diff
@@ -0,0 +1,6 @@
+#Before you merge, lint the files with eslint.
+
+#  `brew install eslint`
+#  `brew install npm`
+#  `npm i`
+#  `eslint --fix *.js`
```

Comment:
does it make sense to either:
- tie this in to a pre-commit local hook?, or
- make the build (GHA) fail if these are not run?

Reply:
Likely yes. These are very outdated scripts that I left in there for now, but i am not sure if we even need it. They might be gone in a sprint or two once I am done setting this up on a BEE 🤞